functional_tests/feature-flags.test.ts

```diff
@@ -1,12 +1,22 @@
-/* eslint-disable compat/compat */
-import { v4 } from 'uuid'
```

Comment:
uuid package is incompatible in some way with jest 28+ and we don't need it, we only use it in this file, so we can uuidv7 instead